packages/radix-ui-themes/src/components/checkbox-card-group.css

```diff
@@ -94,7 +92,6 @@
       --checkbox-card-group-item-padding-bottom: calc(var(--space-5) / 1.2);
       --checkbox-card-group-item-padding-left: var(--space-5);
       --checkbox-card-group-item-border-radius: var(--radius-4);
-      --checkbox-card-group-item-gap: var(--space-4);
```

Comment:
Interesting, that's quite a big change. What's the rationale?

Reply:
Just looks quite bad with a larger gap